web/viewer.css

```diff
@@ -128,40 +130,6 @@
     --overlay-button-hover-color: rgba(115, 115, 115, 1);
 
     --loading-icon: url(images/loading-dark.svg);
```

Comment:
The approach used for the buttons unfortunately doesn't seem to work here, since this one is applied to `<input>` elements which as far as I can tell doesn't support using `::before`/`::after`.

Most likely there's a clever work-around, but so far I've been unable to come up with anything here.
Given that this patch improves things overall, with respect to the size of the image resources in the default viewer, do we perhaps want to consider this patch even as-is?

Reply:
I'm fine with that, but in that case it would be good to add a comment to this one left-over dark image stating why it's still there.

Reply:
Really good point, I've added a comment about this; thank you!